extensions/hibernate-validator/runtime/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <artifactId>quarkus-hibernate-validator</artifactId>
     <name>Quarkus - Hibernate Validator - Runtime</name>
-    <description>Validate data coming to your REST endpoints</description>
+    <description>Validate data (POJO, methods parameters) with built-in integrations for REST, CDI, JPA, ...</description>
```

Comment:
Wdyt about "Offer fields and methods parameters validation for your beans (REST, CDI, JPA, ...)"

Reply:
I went for "Validate object properties (field, getter) and method parameters for your beans (REST, CDI, JPA)" but I think it undersells it.